Ducky-Chickens/bangazonCLI
# Hp data

# Description
Adds PaymentType Model and Controller, adds functionality to CLI prompt chain

## Related Ticket(s)
#3 

## Problem to Solve
Enables user to add a payment type to an active customer

## Proposed Changes
Changes PaymentType data to include limited selection of generic payment 'types'.
Adds case 3 to switch statement in ui.js

## Expected Behavior
User should be able to add an entry to the payment_types table or is sent back to the main menu if no customer is active

## Steps to Test Solution
```npm run db:reset```
```bangazon```

- successfully choose an active customer
- select Create a payment option
- follow the steps to add a payment method
_invalid input should be denied_
_customer id of 8 is currently out of range, fix coming from Tim_

## Testing

[ ] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[ ] I certify that all existing tests pass

## Documentation
[ ] There is new documentation in this pull request that must be reviewed..
[ ] I added documentation for any new classes/methods

## Deploy Notes
